lib/vsc/install/shared_setup.py

```diff
@@ -144,13 +144,19 @@ def _log(self, level, msg, args):
 
 RELOAD_VSC_MODS = False
 
-VERSION = '0.9.10'
+VERSION = '0.9.12'
```

Comment:
This is adding features, so it's a minor version; not a bug fix bump. As it's <0.* I guess all bets are off on the actual versioning scheme.